manifests/y/yuk7/ArchWSL/24.4.28.0/yuk7.ArchWSL.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: yuk7.ArchWSL
+PackageVersion: 24.4.28.0
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: Arch.exe
+InstallModes:
+- silent
+- silentWithProgress
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.WSL

```

```suggestion
  WindowsFeatures:
  - Microsoft-Windows-Subsystem-Linux
```
I forgot to hit "Start a revew".